cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -247,4 +257,15 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
             outputDir.resolve(relativePath.toString()).safeDeleteRecursively()
         }
     }
+
+    private fun archiveAll() {
+        val zipFile = createTempFile(ORT_NAME, ".zip").toFile()
+        zipFile.deleteOnExit()
+
+        outputDir.packZip(zipFile, overwrite = true)
```

Comment:
We shouldn't unconditionally pass `overwrite = true` here as we also don't to that in `archive()`.

Reply:
I guess you are right, but omitting the `overwrite = true` parameter creates an error. This is due to the temp file that has already been created when the `packZip` method is called. Apparently, even this files may not exist if no overwriting is allowed.

Reply:
Not using a temp file, makes this unnecessary.